src/test/java/com/cloudant/tests/ReplicationTest.java

```diff
@@ -103,33 +102,21 @@ public void replication_filteredWithQueryParams() {
     }
 
     @Test
-    public void replicatorDB() throws Exception {
+    public void replicationDB() throws Exception {
         String version = account.serverVersion();
         if (version.startsWith("0") || version.startsWith("1.0")) {
             return;
         }
 
         // trigger a replication
-        Response response = account.replicator()
-                .source(db1URI)
-                .target(db2URI).continuous(true)
-                .createTarget(true)
-                .save();
-
-        // we need the replication to finish before continuing
-        Utils.waitForReplicatorToStart(account, response.getId());
-
-
-        // find all replicator docs
-        List<ReplicatorDocument> replicatorDocs = account.replicator()
-                .findAll();
-        assertThat(replicatorDocs.size(), is(not(0)));
+        ReplicationResult result = account.replication().source(db1URI)
+                .target(db2URI).createTarget(true).trigger();
 
-        Utils.removeReplicatorTestDoc(account, response.getId());
+        assertTrue("The replication should complete ok", result.isOk());
```

Comment:
Is it possible to use `findDocumentWithRetries` to check that all of the documents have replicated successfully to the remote side?